vcs/src/main/java/org/openjdk/skara/vcs/git/GitRepository.java

```diff
@@ -649,6 +649,16 @@ public void push(Tag tag, URI uri, boolean force) throws IOException {
         }
     }
 
+    @Override
+    public void push(String refspec, URI uri, boolean force) throws IOException {
+        if (force && !refspec.equals("+")) {
```

Comment:
Are you sure you did not mean 
```suggestion
        if (force && !refspec.beginsWith("+")) {
```
?

Reply:
I would have opted to just skip the `boolean force` parameter. If we are already entrusting the user to supply a correct refspec then they will probably know that they should prefix the refspec with a `+` if they want to force push.

Reply:
You are right, I meant beginsWith. But having thought about it, I think Erik has a point. We shouldn't modify the refspec given by the user.